src/button.js

```diff
@@ -99,6 +99,9 @@ Button.prototype.getTemplateContext = function () {
 
 Button.prototype.render = function () {
     View.prototype.render.call(this);
+    if (this.ariaLabel && !this.opts.buttonUrl) {
```

Comment:
The button object gets used in non-button instances in the content footer. If there isn't a URL we aren't really a button and doesn't need the label. This was one of the cases where I was pull my hair out over SDK stuff and also that input, share and ui are separate projects because they really seem tightly integrated into core logic.